css/color-scheme.css

```diff
@@ -95,6 +23,21 @@ body {
 .flex { display: flex; gap: var(--gap, 1rem);}
 .grid { display: grid; gap: var(--gap, 1rem);}
 
+/* theme toggle button */
+
+#theme-toggle-button {
+    position: absolute;
+    top: 0;
+    right: 0;
+    height: 40px;
+    width: 40px;
+    background-color: transparent;
```

Comment:
could have used background instead of writing background-color and background -repeat separately